sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/SonarScannerInstaller.java

```diff
@@ -76,9 +86,10 @@ void doInstall(Version scannerVersion, File toDir) {
     }
   }
 
-  private File locateZip(Version scannerVersion) {
+  private File locateZip(Version scannerVersion, @Nullable String classifier) {
     File zipFile = null;
-    URL zip = SonarScannerInstaller.class.getResource("/com/sonar/orchestrator/build/sonar-scanner-" + scannerVersion.toString() + ".zip");
+    String cl = (classifier == null) ? "" : "-" + classifier;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS864)